src/model/table.py

```diff
@@ -0,0 +1,35 @@
+import os
+from model.table_metadata import TableMetaData
+
+
+class Table:
+
+    def __init__(self, database_name, table_map, path):
+        self.table_metadata = TableMetaData(database_name, table_map, path)
+
+    def serialize(self):
+        self.__create_table__()
+        self.__create_indices__()
```

Comment:
You shouldn't create indices here. u create indices in metadata/.
you should  `to metadata.serialize` will write metadata and serialize indices